app/controllers/container_node_controller.rb

```diff
@@ -7,6 +7,10 @@ class ContainerNodeController < ApplicationController
   after_action :cleanup_action
   after_action :set_session_data
 
+  def show_list
+    process_show_list(:where_clause => 'container_nodes.deleted_on IS NULL')
```

Comment:
Well, #1526 sounds like `:where_clause` on `deleted_on` didn't work either?  Unless there's something special about nodes, let's get all 5 consistent, merge this, and debug separately.